### PR TITLE
chore: use "ci" for Dependabot commit message

### PR DESCRIPTION
**Pull request by nschonni**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:

#### What changes did you make? (Give an overview)

Noticed in https://github.com/eslint/eslint/pull/16350 that the commit message formatting check wasn't happy with the default message that Dependabot uses, so changed it so that it should be compliant for those updates to the GitHub Actions dependencies

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#commit-message

### CODE REVIEW

**Review by snitin315 — APPROVED** [review]
This change makes sense to me. I Would like others to review it as well.

**Comment by kecrily** [comment]
I think dependency changes should be `chore` instead of `ci`.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,3 +4,5 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    commit-message:
+      prefix: "ci"
```
Comment (kecrily) [inline .github/dependabot.yml:8]: I think dependency changes should be `chore` instead of `ci`.

Comment (kecrily) [inline .github/dependabot.yml:8]: I think dependency changes should belong to `chore`, aka current value. `ci` should refer to changes to CI configuration (like this PR), not modifications made by CI.

Comment (snitin315) [reply .github/dependabot.yml:8]: Currently I see depandabot PRs for github-actions only. I suppose this is because we don't have a lockfile.

Comment (kecrily) [reply .github/dependabot.yml:8]: Maybe we can try [renovate]()https://docs.renovatebot.com/?

Comment (snitin315) [reply .github/dependabot.yml:8]: We are exploring it - https://github.com/eslint/eslint/issues/15726

Comment (nzakas) [reply .github/dependabot.yml:8]: I agree with @kecrily. “ci” doesn’t seem appropriate for this type of change.

Comment (nschonni) [reply .github/dependabot.yml:8]: From https://eslint.org/docs/latest/developer-guide/contributing/pull-requests `ci - changes to our CI configuration files and scripts.`.
The dependabot updates to GitHub Actions, are only to CI scripts in the `.github/workflows` folder. If Dependabot is later used for other types of updates like NPM, the default of `chore` would still be used, even with this change

Comment (nzakas) [reply .github/dependabot.yml:8]: Ah fair point! I didn't realize this was scoped to just GitHub Actions. In that case, I agree that `ci` makes sense. :+1:

**Review by nzakas — APPROVED** [review]

**Pull request merged**